From: Werner Koch <wk@gnupg.org>
Date: Sat, 7 May 2016 09:09:04 +0200
Subject: syscfg: Add a powerpc and a tilgegx architecture.

* src/syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h: New.
* src/syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h: New.
* src/Makefile.am (lock_obj_pub): Add them.
--

Debian-bug-id: 823630, 823631
Signed-off-by: Werner Koch <wk@gnupg.org>
---
 src/Makefile.am                                    |  2 ++
 .../lock-obj-pub.powerpc-unknown-linux-gnuspe.h    | 23 ++++++++++++++++++++
 src/syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h | 25 ++++++++++++++++++++++
 3 files changed, 50 insertions(+)
 create mode 100644 src/syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h
 create mode 100644 src/syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h

diff --git a/src/Makefile.am b/src/Makefile.am
index 1d88608..5623aff 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -63,6 +63,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.powerpc-unknown-linux-gnu.h     \
         syscfg/lock-obj-pub.powerpc64-unknown-linux-gnu.h   \
 	syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h \
+	syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h  \
         syscfg/lock-obj-pub.s390x-ibm-linux-gnu.h           \
         syscfg/lock-obj-pub.sh4-unknown-linux-gnu.h         \
         syscfg/lock-obj-pub.sparc-unknown-linux-gnu.h       \
@@ -71,6 +72,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.x86_64-pc-linux-gnu.h           \
         syscfg/lock-obj-pub.x86_64-pc-linux-gnux32.h        \
         syscfg/lock-obj-pub.x86_64-pc-linux-musl.h          \
+	syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h      \
 	syscfg/lock-obj-pub.mingw32.h
 
 
diff --git a/src/syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h b/src/syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h
new file mode 100644
index 0000000..c53413b
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h
@@ -0,0 +1,23 @@
+## lock-obj-pub.powerpc-unknown-linux-gnuspe.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[24];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
diff --git a/src/syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h b/src/syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h
new file mode 100644
index 0000000..b99ea32
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.tilegx-unknown-linux-gnu.h
@@ -0,0 +1,25 @@
+## lock-obj-pub.tilegx-unknown-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[40];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
